tbx/core/models.py

```diff
@@ -104,59 +99,40 @@ class Meta:
 
 
 # Home Page
-class HomePageFeaturedPost(Orderable):
-    page = ParentalKey(
-        "torchbox.HomePage", on_delete=models.CASCADE, related_name="featured_posts"
-    )
-    featured_post = models.ForeignKey(
-        "wagtailcore.Page",
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-        related_name="+",
-    )
-
-    panels = [
-        PageChooserPanel("featured_post", ["blog.BlogPage", "work.HistoricalWorkPage"]),
-    ]
-
-
-class HomePageHeroImage(Orderable):
-    page = ParentalKey(
-        "torchbox.HomePage", on_delete=models.CASCADE, related_name="hero_images"
-    )
-    image = models.ForeignKey(
-        "images.CustomImage",
-        help_text="The hero images will be displayed in a random order.",
-        null=True,
-        blank=False,
-        on_delete=models.CASCADE,
-        related_name="+",
-    )
-
-
 class HomePage(ColourThemeMixin, ContactMixin, SocialFields, Page):
     template = "patterns/pages/home/home_page.html"
-    hero_intro_primary = models.TextField(blank=True)
-    hero_intro_secondary = models.TextField(blank=True)
-    intro_body = RichTextField(blank=True)
-    work_title = models.TextField(blank=True)
-    blog_title = models.TextField(blank=True)
-    clients_title = models.TextField(blank=True)
+    statement = models.TextField(blank=True)
+    introduction = models.TextField(blank=True)
+    partners_block = StreamField(
```

Comment:
As mentioned on slack, I don't think we want to use a streamfield for the partner logos. There will only ever be one set of logos, so these should be added directly as a field on the home page model.

Reply:
@lparsons396 we can use `wagtail.models.Orderable` to achieve this, similar to, say, here:

https://github.com/torchbox/torchbox.com/blob/b2a027daaad33f00f09be8bb7828c3a343ec0760/tbx/work/models.py#L26-L38

https://github.com/torchbox/torchbox.com/blob/b2a027daaad33f00f09be8bb7828c3a343ec0760/tbx/work/models.py#L173